test/src/main/java/org/apache/accumulo/test/ShellServerIT.java

```diff
@@ -1421,21 +1431,24 @@ public void help() throws Exception {
     }
   }
 
+  // TODO - evaluate this test is testing what is expected or history is working.
   @Test
   public void history() throws Exception {
     final String table = name.getMethodName();
 
     ts.exec("history -c", true);
     ts.exec("createtable " + table);
     ts.exec("deletetable -f " + table);
+    // TODO - this may be passing only because method name is history.
```

Comment:
There does appear to be an issue with the history test. Changing the name of the table does result in a test failure. I am not quite sure if it has to do with the history command itself or with the way we search for the expectedString.

Reply:
This issue seems to occur in 1.10 as well. Interestingly, the @test annotation for this test was commented out in the 2.0.0 and 2.0.1 release and was uncommented in #1483.